js_modules/dagit/packages/core/src/assets/AssetEvents.tsx

```diff
@@ -317,6 +315,16 @@ const AssetMaterializationGraphs: React.FC<{
           flexDirection: 'column',
         }}
       >
+        {graphedLabels.length === 0 && (
+          <Box padding={{horizontal: 24, top: 64}}>
+            <NonIdealState
+              icon="linear_scale"
+              title="No numeric metadata"
+              description="Include numeric metadata entries in your materializations and observations to see data graphed by time / partition."
```

Comment:
Would we still toggle between time / partition?  If not, maybe drop the `/ partition`.

Should we toggle based on whether it's an SDA or not?

Also, are we sure we want to reserve the real estate if there's no data to display?

Reply:
Ahh yeah we can conditionally mention the partition part 👍 

I think we do want to reserve the space for the graphs (especially if we move forward with hiding the jobs sidebar on these pages and have lots of real estate). I think it's important that people know they could be getting a lot more value if they send metadata.